modules/openy_features/openy_demo_content/modules/openy_demo_webform/openy_demo_webform.install

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * OpenY Demo WebForm install file.
+ */
+
+/**
+ * Implements hook_uninstall().
+ */
+function openy_demo_webform_uninstall() {
+  // TODO: Show warning message in OpenY components page on uninstall
+  // "Be careful, if you use demo_form webform on site, module
+  // uninstall will remove all submissions and demo_form from site".
+  \Drupal::service('openy.modules_manager')
```

Comment:
It's good to have this code in place but have to mark the module impossible to uninstal. Please add dependency from ```openy``` now and todo there for making this more clear.
After message is ready for the user - we'll remove this hack

Reply:
@podarok It's possible to uninstall the module, we just want to inform the user that disabling this component use will remove demo webforms. Just in case they started to use demo webform as permanent.  We plan to add this message to the custom page that will allow disabling OpenY components.

Reply:
In my comment above I said we must disable uninstall for now, until message is ready @ddrozdik

Reply:
@podarok dependency from openy profile can't prevent module uninstall. So in this case we need add it as dependency to any module from openy.